randovania/patching/prime/all_prime_dol_patches.py

```diff
@@ -78,13 +93,21 @@ def remote_execution_patch_start() -> list[BaseInstruction]:
         *return_code,
     ]
 
-    num_bytes_to_invalidate = _remote_execution_max_byte_count - assembler.byte_count(intro)
+    num_bytes_to_invalidate = _remote_execution_max_byte_count - (assembler.byte_count(cutscene_check) + 8 +
+                                                                  return_code_byte_count) - assembler.byte_count(intro)
     # Our loop end condition depends on this value being a multiple of 32, greater than 0
     num_bytes_to_invalidate = ((num_bytes_to_invalidate // 32) + 1) * 32
 
     return [
         *intro,
 
+        # if camera count > 0 then we're in a cutscene so we return as we don't want
+        # to handle receiving items during a cutscene
+        *cutscene_check,
+        cmpwi(r0, 0),
+        ble(return_code_byte_count + 4, True),
+        *return_code,
```

Comment:
This return code is still duplicated. Instead of having all the return block again, you could just jump to the previous return block.
(The if condition would need to be inverted though).

Reply:
I had to implement bgt